### DISCUSSION TITLE
Cannot ignore a terminology rule on a line of a markdown file that references a specific header name of an excel file

### DISCUSSION THREAD

**Discussion by hepcat72** [textlint/textlint#1585] [Q&A]
**What version of textlint are you using?**

```
$ textlint -v
v14.4.0
```

**What file type (Markdown, plain text, etc.) are you using?**

Markdown

**What did you do? Please include the actual source code causing the issue.**

Lines 70-72 of the file in question:
```
    <!-- textlint-disable terminology -->
    "mzXML File Name", "Sequence", and "Skip" values (the only way to determine
    <!-- textlint-enable -->
```

The occurrence of `mzXML File Name` is a reference to the name of an excel sheet header.  Changing it to "filename" would be invalid, so I'm trying to make textlint ignore it, but nothing I try eliminates the following reported error when I run:

```
$ textlint .github/pull_request_template.md
```

**What did you expect to happen?**

The filename error should be ignored, given the in-line ignore directive.

**What actually happened? Please include the actual, raw output from textlint.**

```
/Users/rleach/PROJECT-local/TRACEBASE/tracebase-rabinowitz-data/.github/pull_request_template.md
  71:12  ✓ error  Incorrect term: “File Name”, use “filename” instead  terminology

✖ 1 problem (1 error, 0 warnings)
✓ 1 fixable problem.
Try to run: $ textlint --fix [file]
```

**Comment by hepcat72** [comment]
I also tried `<!-- textlint-disable -->`

**Comment by azu** [comment]
Can you create reproduce repository?

**Comment by hepcat72** [comment]
I'm not sure I understand your question. It is in a private GitHub repository and this error turned up when I updated superlinter from slim v4 to slim v6.x. I reproduced it on the command line.

Are you asking for a link to the repo?

**Comment by azu** [comment]
It is helpful to have a minimum reproduction repository to debug.


Perhaps a blank line break is needed between the comment and the text.

```
<!-- textlint-disable terminology -->

"mzXML File Name", "Sequence", and "Skip" values (the only way to determine

<!-- textlint-enable -->
```

Also, HTML comments written in code blocks cannot be treated as comments.
If there is a minimal repository to reproduce, it can be investigated.
